examples/datasets/Davis.py

```diff
@@ -24,9 +24,38 @@
     plot_voltages,
```

Comment:
Change `Davis.py` -> `davis.py` to follow PEP8 naming conventions.

Reply:
hmm, it's lowercase on my machine, I changed it last time I pushed. I'll check that out